synapse/storage/util/id_generators.py

```diff
@@ -262,6 +281,34 @@ def manager():
 
         return manager()
 
+    async def get_next_mult(self, n: int):
+        """
+        Usage:
+            with await stream_id_gen.get_next_mult(5) as stream_ids:
+                # ... persist event ...
```

Comment:
To match `StreamIdGenerator`:

```suggestion
                # ... persist events ...
```